### PR TITLE
Feature/{chimereucheya}/service page

### PR DESCRIPTION
**Pull request by Merdoth**
#### What does this PR do?
-Application Service Page

#### Description of Task to be completed?
- Shows the application service page which gives in-depth information on what we do at lifeAid, and the services we can offer.

#### How should this be manually tested?
- run the application using `npm run dev` visit `http://localhost:3000/service`

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
- story type: Feature
- story title: Service Page

#### Screenshots (if appropriate)
N/A

#### Question
N/A

### CODE REVIEW

File: components/Service/ServicePage.js [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+            <div className="services app-content">
+                <div className="content-wrapper services-wrapper service-wrap">
+                    <div className="services-content">
+                        <img className="serviceI" src="/static/img/help.jpg" alt="Smiley face" />
+                        <span className="services-content-text">
+                            <p className="service-text1">
+                                Emergency Contact
+                                <br />
+                                {' '}
+                                Number
```
Comment (Way2nnadi) [inline components/Service/ServicePage.js:130]: What is this `Number` doing here?

Comment (Merdoth) [reply components/Service/ServicePage.js:130]: I don't understand your question

File: components/Service/ServicePage.js [hunk 2]
```diff
@@ -0,0 +1,172 @@
+/* eslint-disable react/jsx-indent-props */
+import React from 'react';
+import Head from 'next/head';
+import NavigationBar from '../Common/NavigationBar';
+import './service.scss';
+import { HOME_STRINGS } from '../constants';
+import { SERVICE_STRING } from './constants';
```
Comment (EngrEric) [inline components/Service/ServicePage.js:7]: separation of concerns is important. move strings for this component to same place

File: components/Service/ServicePage.js [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+
+const {
+    GOOGLE_FONTS,
+    INTRO_TEXT2,
+} = SERVICE_STRING;
```
Comment (EngrEric) [inline components/Service/ServicePage.js:12]: `const {
    GOOGLE_FONTS,
    INTRO_TEXT2,
} = SERVICE_STRING;`

File: components/Service/ServicePage.js [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+                <hr />
+                <p className="intro-header">{SERVICES}</p>
+                <div className="intro-service-wrapper">
+                    <div className="content-wrapper services-wrapper intro-wrap">
+                        <div className="intro-content">
+                            <i className="fas fa-users icons-status" />
+                            <hr />
+                            <span className="services-content-text context1">
+                                <p className="intro-text1">
+                                   Online Counselling
```
Comment (EngrEric) [inline components/Service/ServicePage.js:61]: move strings to constant

**Review by EngrEric — CHANGES_REQUESTED** [review]
I will suggest you use UI library like `ant d` to make your work cleaner

File: components/Service/ServicePage.js (hunk 1 above)
Comment (Way2nnadi) [reply components/Service/ServicePage.js:130]: You have `Number` on line 30, what is its reason for being there?

File: components/Service/ServicePage.js (hunk 2 above)
Comment (Merdoth) [reply components/Service/ServicePage.js:7]: Like you said I should practice DRY so I can't be repeating the same string that already exists when I can just import it from the initial place and if you pay enough attention to details you'll notice that those two constants are serving two very different purpose

File: components/Service/ServicePage.js (hunk 4 above)
Comment (Merdoth) [reply components/Service/ServicePage.js:61]: I don't see the need to move a non-reusable string that is less than 20 characters into a constant file. That's why you can see those few strings around

**Comment by Merdoth** [comment]
> I will suggest you use UI library like `ant d` to make your work cleaner

Whether I use `antd` or not it actually doesn't make much difference and besides, If the purpose of my installing UI library like `antd` is to make my work cleaner then what exactly is the use of eslint?

**Pull request merged**